msnoise/s000installer.py

```diff
@@ -78,42 +88,92 @@ def create_database_inifile(tech, hostname, database, username, password,
     f.close()
 
 
-def main(tech=None, hostname="localhost", username="msnoise",
-         password="msnoise", database="msnoise", filename="msnoise.sqlite",
-         prefix=""):
-    global install_mode
-    install_mode = True
+def create_indices(session):
+    """
+    Create indices in the database, using the provided sqlalchemy session.
+    """
+    # TODO move those calls to a def and call it from install / msnoise.scripts
+    try:
+        session.execute("CREATE UNIQUE INDEX job_index ON %sjobs (day, pair, "
+                        "jobtype)" % prefix)
```

Comment:
Xavier: ``prefix`` will not be know here ?

Reply:
Of course, it won't :/ I overlooked this, thanks. I apparently did not do enough tests with the database after my later changes before pushing, sorry.